NewsApp/NewsApp/Services/APIService.swift

```diff
@@ -17,9 +17,13 @@ enum APIResult<T, Int> {
     case Failure(Error)
 }
 
-class APIService {
+class APIService: DataLoader {
+    
+    var apiServiceDelegate: DataLoaderDelegate?
```

Comment:
weak

Reply:
weak' must not be applied to non-class-bound 'DataLoaderDelegate'; consider adding a protocol conformance that has a class bound